src/internal/Subscriber.ts

```diff
@@ -62,7 +62,9 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
    * @return {void}
    */
   next(value?: T): void {
-    if (!this.isStopped) {
+    if (this.isStopped) {
+      handleStoppedNotification({ kind: 'N', value }, this);
+    } else {
       this._next(value!);
     }
```

Comment:
Any registered function would be called (asynchronously) within the `next`, `error` and `complete` methods of `Subscriber`.